lib/nsxv.py

```diff
@@ -614,10 +615,11 @@ def __str__(self):
         term_str = str(term)
         if term_str:
           target.append(term_str)
-      target.append('\n')
 
       # ensure that the header is always first
       target = target_header + target
       target.append('%s' % (_XML_TABLE.get('sectionEnd')))
       target.append('\n')
-    return '\n'.join(target)
+
+      target_as_xml = xml.dom.minidom.parseString(''.join(target))
+    return target_as_xml.toprettyxml()
```

Comment:
I would add the option indent="  " to the toprettyxml(), that removes the tab and makes it two spaces. (taken from: https://docs.python.org/2/library/xml.dom.minidom.html)